amqp/amqp.go

```diff
@@ -626,18 +627,53 @@ func fromDelivery(d *amqp.Delivery) (*queue.Job, error) {
 	j.Raw = d.Body
 
 	if retries, ok := d.Headers[DefaultConfiguration.RetriesHeader]; ok {
-		retries, ok := retries.(int32)
-		if !ok {
-			return nil, ErrRetrievingHeader.New(DefaultConfiguration.RetriesHeader, d.MessageId)
-		}
+		switch r := retries.(type) {
+		case int16:
```

Comment:
Why this case for `int16?`

Reply:
The three integer formats supported by amqp library are `int16`, `int32` and `int64`. There for completeness. It says that supports `int` but returns error if it is used.

https://godoc.org/github.com/streadway/amqp#Table